provider/uprops/src/binary.rs

```diff
@@ -49,9 +49,9 @@ impl<'data> DataProvider<'data, UnicodePropertyV1Marker> for BinaryPropertiesDat
             metadata: DataResponseMetadata {
                 data_langid: req.resource_path.options.langid.clone(),
             },
-            payload: Some(DataPayload::from_owned(UnicodePropertyV1::from_uniset(
```

Comment:
Done. Similar to what I needed to do for the getter functions in components/uniset's `props.rs`, I changed the unit test helper fn to return the DataPayload rather than an owned UnicodeSet since the DataPayload and UnicodeSet it returns now have the `'data` lifetime parameter